website/src/components/MenuTitleChildren/MenuTitleChildren.jsx

```diff
@@ -36,17 +35,22 @@ const HandleChildRenderEvents = (props) => {
 }
 
 const MenuTitleChildren = (props) => {
-    const children = []
-    useEffect(()=>{}, [props.items])
-    props.items.forEach((child, index) => {
-        children.push(
-           <HandleChildRenderEvents key={index} path={child.path} name={child.name} index={index} onClick={props.onClick} onDeleteClick={props.onDeleteClick} showDelete={props.showDelete}/>
+    const [children, setChildren] = useState([]);
```

Comment:
Does this component has no `props.children`? 
What component is it, Please can you attach a screenshot?

Reply:
the component that renders the title and the list of children, on the right-hand side of the menu, for the` props.children` you had suggested to rename it to `items`

Reply:
![image](https://user-images.githubusercontent.com/98592125/177474877-50341e6b-91c6-4cf2-b7ef-071c9c2dcb6e.png)
this component this single title and children